src/send_status.py

```diff
@@ -3,8 +3,6 @@
 
 import utils
 
-from dlc.github_handler import GitHubHandler
```

Comment:
this file should not be executed when running local builds. any reason for changing?

Reply:
I just changed all files where GitHubHandler is imported, indiscriminately.

I can revert the change in this file. I'll do this and push another commit, if there are no other comments in your review.